### ISSUE TITLE
DVB-T / Blog V4

### ISSUE THREAD

**Issue by MiggyAjuice** [rtlsdrblog/rtl-sdr-blog#50]
So I know that the new Blog V4 requires new drivers to get SDR functionality, so everytime I install normal DVB-T drivers to watch some Digital TV, I never get any reception like I used to with my now broken old blue SDR dongle. Is there a possible workaround for the new Blog V4 as well with DVB? Otherwise, the Blog V4 is a really enjoyable device when it comes to SDR, I've been enjoying using it a lot.

**Comment by dwadi** [comment]
Have you tested it via Android app?

**Comment by MiggyAjuice** [comment]
Yep, I assume drivers for the specific Blog V4 haven't been tested on Android either. When I try on Android, it says something along the lines of "The USB OTG is not giving enough power" something like that. Didn't say that when I had my old now broken blue no-name. The android driver developer said last year they would try to get the Blog V4 working by earliest February 2024 so I assume it's going to be in the works anytime soon.

**Comment by MiggyAjuice** [comment]
On reddit ive also received a reply to implement a backport of the patch code to video4linux but things got too technical
